integration/admin-client/src/main/java/org/keycloak/admin/client/resource/ClientTemplateResource.java

```diff
@@ -44,7 +45,7 @@
 
     @PUT
     @Consumes(MediaType.APPLICATION_JSON)
-    public void update(ClientTemplateRepresentation rep);
+    public Response update(ClientTemplateRepresentation rep);
```

Comment:
We shouldn't change this to return a Response as that requires closing the response and there is no need for it. Rather just rely on exceptions like the other update methods do.